lib/util.js

```diff
@@ -230,17 +230,30 @@ function format(f) {
   return str;
 }
 
-var debugs = {};
-var debugEnviron;
+const debugs = {};
+let debugEnviron;
+
+const regExpSpecialChars = /[|\\{}()[\]^$+?.]/g;
+
+function arrayToRegExp(strArr) {
+  // Escape special chars except wildcard.
+  strArr = strArr.map((str) => {
+    return str.replace(regExpSpecialChars, '\\$&')
+      .replace(/\*/g, '.*');
+  });
+
+  return new RegExp(`^${strArr.join('|')}$`, 'i');
```

Comment:
Instead of calling `map` and `join` you could use `reduce`
```js
const regExpStr = strArr.reduce((str, entry, i) => {
  let part = entry.replace(regExpSpecialChars, '\\$&')
    .replace(/\*/g, '.*');
  if (i -1 !== strArr.length)
    part += '|';
  return `${str}${part}`;
}, '');
return new RegExp(`^${regExpStr}$`, 'i');
```
But that might just be my personal preference.

Reply:
I think the current code is a bit more readable (not having to deal with the indices is nice, at least :))